R/get_kr_comp.R

```diff
@@ -23,27 +23,28 @@ h_get_kr_comp <- function(tmb_data, theta) {
 #' fit <- mmrm(FEV1 ~ ARMCD + ar1(AVISIT | USUBJID), data = fev_data)
 #' contrast <- matrix(c(0,1), ncol = 1)
 #' kr(fit, contrast)
-kr <- function(fit, contrast) {
+kr <- function(fit, contrast, order = 2L) {
```

Comment:
what is `order`? do we support another order than 2?

Reply:
KR (linear); I am adding these tests currently

Reply:
But then we just have order one or two right? Then I would say better make it a flag instead of a numeric argument

Reply:
user `linear` as a flag argument